infrastructure/quick-deploy/aws/parameters.tfvars

```diff
@@ -410,6 +447,17 @@ fluent_bit = {
   node_selector = {}
 }
 
+#  Uncomment the following fluent_bit_windows to deploy fluent_bit on windows node group
+# fluent_bit_windows = {
+#   is_daemonset = true
+#   #image_name   = "fluent/fluent-bit"
+#   image_tag = "windows-2022-3.2.0"
+#   node_selector_windows = {
+#     "platform" = "windows"
+#   }
+# }
```

Comment:
I think this could always be uncommented. It is not an error to have a daemon set without any node.